var/spack/repos/builtin/packages/asagi/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+# Author: Gilbert Brietzke
+# Date: July 2, 2019
+
+from spack import *
+
+
+class Asagi(CMakePackage):
+    """a pArallel Server for Adaptive GeoInformation."""
+
+    homepage = "https://github.com/TUM-I5/ASAGI"
+    git = "https://github.com/TUM-I5/ASAGI.git"
+
+    version('f633f96', commit='f633f96931ae00805f599078d5a1a6a830881554',
+            submodules=True, preferred=True)
```

Comment:
Please add the official version `1.0` as a tarball download:
```
https://github.com/TUM-I5/ASAGI/archive/v1.0.tar.gz
```
I assume there's a reason you require the latest version instead of the official release?  Please add it as a `master` version (search for other packages with `version('master'...`).  Put the `master` version before the `1.0` version.

Alternately, keep the version you have, but rename it to a "made-up" interim version number that sorts correctly: i.e. `1.0.1`.  This is more secure than providing a `master` version, and is guaranteed not to change.

Reply:
there seem to be issues with when compiling @1.0 with gcc@5:
however - I put in a conflict and have version 1.0 and most recent commit as version 1.0.1 (as suggested)
it is helpfull to checkout the repository to have included a git-submodule, not included within the tar